fabric-network/src/events.ts

```diff
@@ -24,6 +24,7 @@ export interface TransactionEvent {
 	readonly transactionData: fabproto6.protos.ITransaction | fabproto6.protos.IFilteredTransaction;
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	readonly privateData?: any;
+	readonly timestamp?:string;
```

Comment:
It might be more helpful for us to expose a JavaScript `Date` object instead of a string